README.md

```diff
@@ -2,4 +2,14 @@
 
 TuneUp is a view extension for analyzing the performance of Dynamo graphs. TuneUp allows you to see overall graph execution time, per-node execution time, and other helpful information about what's happening under the hood.
 
-![Alt text](design/images/TuneUp_Mockup_03_SortedByExecutionOrder.jpg?raw=true "TuneUp")
\ No newline at end of file
+![Alt text](design/images/TuneUp_Mockup_03_SortedByExecutionOrder.jpg?raw=true "TuneUp")
+
+## Testing
+
+### Setup
+
+- Download DynamoCoreRuntime 2.5 from https://dynamobuilds.com/
```

Comment:
hmm - so any chance of doing this automatically? 
maybe we can add another task to automate this since it seems we should put the runtime on nuget.

Reply:
I'd prefer to work on this as a separate PR/task in which we look at automating the whole test setup (including dealing with TuneUp dll and manifest locations). I'd like to get this PR merged asap--its been open for a while and contains a lot already.